qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -9,6 +9,14 @@ The set of required fields for the *sample information file* and *prep
 information file* files varies based on the functionality that you want to
```

Comment:
Can you replace for: "The set of required fields for the *sample information* and *preparation information files*?